### PR TITLE
feat: title decomposition

### PR DESCRIPTION
**Pull request by PetrochukKsenija**

### CODE REVIEW

File: translated_scs/title_part/src/toc.scs [hunk 1]
```diff
@@ -19,4 +20,141 @@
 	<- lang_ru;;
 	=> nrel_format: format_html;;
 *);
-=> nrel_note: [<p>Названия тех <i>разделов</i>, которые планируется написать впоследующих изданиях <i>Стандарта OSTIS</i> или <i>разделов</i>, которые<u>в данном</u> издании <i>Стандарта OSTIS</i> не печатаются, посколькусодержание их не изменилось по сравнению с предыдущим <u>явно указываемым</u>изданием <i>Стандарта OSTIS</i>, выделяются также <u>жирным курсивом</u>, нодля них страницы в рамках данного издания <i>Стандарта OSTIS</i> неуказываются</p>];;
+=> nrel_note: [<p>Названия тех <i>разделов</i>, которые планируется написать впоследующих изданиях <i>Стандарта OSTIS</i> или <i>разделов</i>, которые<u>в данном</u> издании <i>Стандарта OSTIS</i> не печатаются, посколькусодержание их не изменилось по сравнению с предыдущим <u>явно указываемым</u>изданием <i>Стандарта OSTIS</i>, выделяются также <u>жирным курсивом</u>, нодля них страницы в рамках данного издания <i>Стандарта OSTIS</i> неуказываются</p>]
```
Comment (MakarenkoAI) [inline translated_scs/title_part/src/toc.scs:23]:
```suggestion
=> nrel_note: [<p>Названия тех <i>разделов</i>, которые планируется написать в последующих изданиях <i>Стандарта OSTIS</i> или <i>разделов</i>, которые<u>в данном</u> издании <i>Стандарта OSTIS</i> не печатаются, поскольку содержание их не изменилось по сравнению с предыдущим <u>явно указываемым</u> изданием <i>Стандарта OSTIS</i>, выделяются также <u>жирным курсивом</u>, но для них страницы в рамках данного издания <i>Стандарта OSTIS</i> не указываются</p>]
```

**Review by MakarenkoAI — APPROVED** [review]

File: translated_scs/title_part/src/toc.scs (hunk 1 above)
Comment (MksmOrlov) [reply translated_scs/title_part/src/toc.scs:23]: это надо и в tex файле пофиксить

**Review by MksmOrlov — APPROVED** [review]

**Review by MksmOrlov — APPROVED** [review]
а why do you create merge commits?

**Comment by PetrochukKsenija** [comment]
There was a conflict in the file with specification of the standard. I resolve the conflict before merging

**Pull request merged**